miscellaneous/build_templates/analysis.yml

```diff
@@ -0,0 +1,190 @@
+parameters:
+  language: ''
+  lib: false
+  skipBinSkim: false # Tells the analysis pipeline not to run binskim as it doesn't work for some samples
+  deleteBinSkim: '' # Some DLLs cause BinSkim errors and need to be removed before running
```

Comment:
I would just say `files` as it's not only DLLs. I'd also clarify this is only used when the errors are investigated and found to be extraneous, something like `This parameter is used delete files before running BinSkim, only use this if the error has been investigated and can be safely ignored`